pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -6849,14 +6849,14 @@ struct pjsua_media_config
     /**
      * Jitter buffer minimum prefetch delay in msec.
      *
-     * Default: -1 (to use default stream settings, currently 60 msec)
+     * Default: -1 (to use default stream settings, currently codec ptime)
```

Comment:
currently is 1, I believe.
https://github.com/pjsip/pjproject/blob/master/pjmedia/src/pjmedia/stream.c#L2694

Reply:
Yes, because 1 = stream->codec_param.info.frm_ptime/stream->codec_param.info.frm_ptime.
But I preferred using time related terms (and units) for consistency with the other values and minimal impact. I agree that it is somehow confusing.